content/actions/hosting-your-own-runners/removing-self-hosted-runners.md

```diff
@@ -51,7 +51,7 @@ To remove a self-hosted runner from an organization, you must be an organization
 
 {% data reusables.organizations.navigate-to-org %}
 {% data reusables.organizations.org_settings %}
-{% data reusables.repositories.settings-sidebar-actions %}
+{% data reusables.github-actions.settings-sidebar-actions-runners %}
 {% data reusables.github-actions.self-hosted-runner-removing-a-runner %}
 
 ### Removing a runner from an enterprise
```

Comment:
I think we still have a missing step [here](https://docs-6161--patch-1.herokuapp.com/en/actions/hosting-your-own-runners/removing-self-hosted-runners#removing-a-runner-from-an-enterprise) to click the Runners tab.

Reply:
Fixed, thank you!